### PR TITLE
Document that llvm-lit needs to be run to collect static metrics

### PR DESCRIPTION
**Pull request by hiraditya**

### CODE REVIEW

**Comment by v01dXYZ** [comment]
Yes, actually `compile_time` metrics has already produced by running make/ninja and `lit` is just aggregating the `.o.time` per directory. Cf litsupport/modules/compiletime

Just to ask because I didn t find it, how to have a repeated build to have at least a averaged `compile_time` ? When we use perf there is the `-r` option but the wrapper script timeit.sh don t allow to add any options.

File: llvm/docs/TestSuiteGuide.md [hunk 1]
```diff
@@ -266,7 +269,7 @@ Example usage:
 
   ```bash
   % test-suite/utils/compare.py -m compile_time baseline.json
-  % test-suite/utils/compare.py -m size.__text baseline.json
+  % test-suite/utils/compare.py -m size..text baseline.json
```
Comment (ilovepi) [inline llvm/docs/TestSuiteGuide.md:272]: Not sure this should change. Apparently, they’re both correct, depending on the platform. https://github.com/llvm/llvm-test-suite/blob/47a750394dfa29d10cdb515132a525d3e8fa3118/litsupport/modules/codesize.py#L41

**Review by ilovepi — APPROVED** [review]
LGTM modulo the one comment.

File: llvm/docs/TestSuiteGuide.md (hunk 1 above)
Comment (hiraditya) [reply llvm/docs/TestSuiteGuide.md:272]: i see. i'll remove this.

**Comment by hiraditya** [comment]
> Yes, actually `compile_time` metrics has already produced by running make/ninja and `lit` is just aggregating the `.o.time` per directory. Cf litsupport/modules/compiletime
> 
> Just to ask because I didn t find it, how to have a repeated build to have at least a averaged `compile_time` ? When we use perf there is the `-r` option but the wrapper script timeit.sh don t allow to add any options.

Does this help:
https://llvm.org/docs/TestSuiteGuide.html#displaying-and-analyzing-results

**Comment by v01dXYZ** [comment]
@hiraditya sorry i was tryharding I should have open an issue instead. Running lit don t rebuild anything. So the result for compile time are noisy and there is no way to estimate the variance and know if the number is statically significant.
That's quite a PITA for sharing reliable results to others.

**Comment by hiraditya** [comment]
> @hiraditya sorry i was tryharding I should have open an issue instead. Running lit don t rebuild anything. So the result for compile time are noisy and there is no way to estimate the variance and know if the number is statically significant. That's quite a PITA for sharing reliable results to others.

yeah please open a separate issue.

**Pull request merged**